conda_deps/conda_deps.py

```diff
@@ -244,7 +244,8 @@ def scan_r_imports(filename):
     with open(filename) as f:
         data = f.read()
 
-    results = re.findall(r"library\((\W*)(\w*)(\W*)\)", data)
+    results = re.findall(r"library\((\W*)([\w,.]+)(\W*)\)", data)
```

Comment:
Thanks Adam, do you mean the following?
```suggestion
    results = re.findall(r"library\((\W*)([\w\.]+)(\W*)\)", data)
```

Reply:
@sebastian-luna-valero funnily enough either actually work, but your correct I meant the backslash (just got a new keyboard).